bravetechnologycoop/BraveSensor
# Text message updates (CU-271q62d, CU-25e7zqf)

## Test Plan
- Deploy to dev
- Run smoketests
   - Messages do not include "bathroom" and still make sense
- Use postman to send a low battery signal to the API for server-side state machine (/api/door)
   - See message including the bit.ly link
   - bit.ly link goes to the correct video when you click on it from a smartphone
   - bit.ly link goes to the correct video when you type in the address
   - Message still makes sense
- Use postman to send a low battery signal to the API for firmware state machine (/api/heartbeat)
   - See message including the bit.ly link
   - bit.ly link goes to the correct video when you click on it from a smartphone
   - bit.ly link goes to the correct video when you type in the address
   - Message still makes sense
